aea/aea.py

```diff
@@ -175,9 +176,12 @@ def _handle(self, envelope: Envelope) -> None:
         logger.debug("Handling envelope: {}".format(envelope))
         protocol = self.resources.protocol_registry.fetch(envelope.protocol_id)
 
-        error_handler = self.resources.handler_registry.fetch_by_skill(
-            "default", "error"
+        # TODO make this working for different skill/protocol versions.
```

Comment:
for now, we are using the full public id for all the functionalities. However, might be desirable in some contexts to just refer to the pair `author/package_name`, since only one version will be active at one time.